Packs/CommonScripts/Scripts/CVECVSSColor/CVECVSSColor.py

```diff
@@ -0,0 +1,58 @@
+
+from CommonServerPython import *
+from typing import Union
+import json
+import math
+
+
+def get_color(cvss: Union[int, float]) -> str:
+    """
+    Gets a CVSS score as an integer\float and sends back the correct hex code for a color as a string.
+
+    Args:
+        cvss (int\float): A CVE CVSS score.
+
+    Returns:
+        str: The color of the score in hex format
+    """
+
+    colors = {
+        'Green1': '#50C878',
+        'Green2': '#6CB65B',
+        'Green3': '#89C35B',
+        'Green4': '#A3C157',
+        'Amber1': '#FFB347',
+        'Amber2': '#FFA07A',
+        'Amber3': '#FF7F50',
+        'Red1': '#FF6347',
+        'Red2': '#FF4500',
+        'Red3': '#FF4040'
+    }
+
+    cvss = int(math.ceil(cvss))
+
+    if not 0 < cvss <= 10:
+        color = "#000000"
+    elif cvss <= 4:
+        color = colors[f"Green{cvss}"]
+    elif cvss <= 7:
+        color = colors[f"Amber{cvss - 4}"]
+    else:
+        color = colors[f"Red{cvss - 7}"]
+
+    return color
+
+
+def main():
+    indicator = demisto.callingContext['args']['indicator']
+
+    cvss = indicator.get('CustomFields').get('cvss', '')
```

Comment:
If ```CustomFields``` do not exist in the indicator, get will result in ```None```
which will result an error when getting cvss.

Reply:
Which will put `***` in the section, thats fine (but i can change if you want me to)

Reply:
It will fail with an error, as you're trying to get the cvss field from None.